src/web/pages/reports/row.js

```diff
@@ -106,7 +106,9 @@ const Row = ({
   if (isDefined(task)) {
     if (task.isContainer()) {
       status =
-        status === TASK_STATUS.running
+        status === TASK_STATUS.interrupted
+          ? TASK_STATUS.uploadinginterrupted
+          : status === TASK_STATUS.running || status === TASK_STATUS.processing
           ? TASK_STATUS.uploading
           : TASK_STATUS.container;
```

Comment:
```suggestion
            ? TASK_STATUS.uploading
            : TASK_STATUS.container;
```
Could you add a level of indentation here to make it more obvious which ternary operators belong together?